docker-compose.yml

```diff
@@ -1,17 +1,17 @@
 # This docker-compose file is used for tests only
 
-version: "3"
+version: '3'
```

Comment:
Our convention so far has been to use double instead of single quotes; we should probably hold to that convention for now.